src/pages/Statistics/StatisticsRegions.tsx

```diff
@@ -228,8 +229,16 @@ const StatisticsCities = () => {
 
     // seting (for chart needs) statisticsItems indicators of the very first element
     // because they are the same for all the elements
+    let entryToSetIndicators = response.data.find((entry: any) => entry.yearStatistics.length != 0)
```

Comment:
maybe it's worth to move that method to some service and avoid duplicating it?

Reply:
> maybe it's worth to move that method to some service and avoid duplicating it?

this method is only used in these two cases, I'm not sure if there's a need for that

Reply:
up to you
but from my point of view, using the exact same two pieces of code in two places - it's duplicating